maxim/logger/crewai/client.py

```
@@ -50,10 +50,24 @@ def get_log_level(debug: bool) -> int:
 
 
 class MaximUsageCallback:
+    """Maxim usage callback.
+
+    This class represents a usage callback.
+    """
+
     def __init__(self, generation_id: str):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)